packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -143,31 +143,10 @@ class _CupertinoTextFieldSelectionGestureDetectorBuilder extends TextSelectionGe
 /// [controller]. For example, to set the initial value of the text field, use
 /// a [controller] that already contains some text such as:
```

Comment:
This sentence should probably be part of the sample description instead of in this part of the API docs.

If you look at the current version on the site, you can see what I mean:  https://api.flutter.dev/flutter/cupertino/CupertinoTextField-class.html

It reads kind of odd, since the example box doesn't have any description at all.
Compare it to another example: https://api.flutter.dev/flutter/cupertino/CupertinoSwitch-class.html and you see that the sample is much more self-contained.

Reply:
That's definitely better with self-contained description, done,